src/components/carousel/Carousel.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+
+import { CarouselData } from "../../data/carousel.data";
+
+const Carousel = () => {
```

Comment:
this is a nice, carousel. I also wanted to implement this in mine. Got an idea from your implementation.